lib/active_record/connection_adapters/sqlserver_adapter.rb

```diff
@@ -44,6 +44,7 @@ class SQLServerAdapter < AbstractAdapter
 
       cattr_accessor :cs_equality_operator, instance_accessor: false
       cattr_accessor :use_output_inserted, instance_accessor: false
+      cattr_accessor :use_output_inserted_into, instance_accessor: false
```

Comment:
Would like to avoid using this and assume use_output_inserted leverages INTO as our default now